### PR TITLE
 Update workshop (bar the Table Viewer) to 0.36.2

### PR DESCRIPTION
**Pull request by leepa**
*Description of changes:*

The TableViewer doesn't presently work, but the rest of the workshop will - this updates the workshop to the latest developer preview (at the time of writing). I've also updated all the samples in the code/ folder as well.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: workshop/content/20-create-project/300-structure.md [hunk 1]
```diff
@@ -36,12 +36,11 @@ You'll see something like this:
 Let's have a quick look at `bin/cdk-workshop.ts`:
 
 ```ts
-import cdk = require('@aws-cdk/cdk');
+import cdk = require('@aws-cdk/core');
 import { CdkWorkshopStack } from '../lib/cdk-workshop-stack';
 
 const app = new cdk.App();
 new CdkWorkshopStack(app, 'CdkWorkshopStack');
-app.run();
```
Comment (dgomesbr) [inline workshop/content/20-create-project/300-structure.md]: Should that be app.synth?

File: code/java/pom.xml [hunk 2]
```diff
@@ -38,41 +38,41 @@
         <!-- AWS Cloud Development Kit -->
         <dependency>
             <groupId>software.amazon.awscdk</groupId>
-            <artifactId>cdk</artifactId>
-            <version>0.22.0</version>
```
Comment (dgomesbr) [inline code/java/pom.xml]: Would be better to export version to <cdk.version>0.36.2.DEVPREVIEW</cdk.version> and ref on it?

File: code/java/src/main/java/com/myorg/CdkWorkshopStack.java [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 
-import software.amazon.awscdk.App;
-import software.amazon.awscdk.Stack;
-import software.amazon.awscdk.StackProps;
-import software.amazon.awscdk.samples.tableviewer.TableViewer;
-import software.amazon.awscdk.samples.tableviewer.TableViewerProps;
+import software.amazon.awscdk.core.App;
+import software.amazon.awscdk.core.Stack;
+import software.amazon.awscdk.core.StackProps;
+// import software.amazon.awscdk.samples.tableviewer.TableViewer;
```
Comment (dgomesbr) [inline code/java/src/main/java/com/myorg/CdkWorkshopStack.java]: Remove comments?

File: workshop/content/20-create-project/300-structure.md (hunk 1 above)
Comment (leepa) [reply workshop/content/20-create-project/300-structure.md]: No this is not needed in the TypeScript version of the CDK as it's implied instead. You can see this from the default template from a `[cdk init](https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/lib/init-templates/app/typescript/bin/%25name%25.template.ts)`.

**Pull request merged**